components/admin-panel/sections/InvoicesReceipts.js

```diff
@@ -65,17 +80,15 @@ const InvoicesReceipts = ({ collective }) => {
   };
 
   const getInvoiceTemplatesObj = () => {
-    if (!alternativeReceiptTitle && !alternativeInfo) {
-      return {
-        templates: {
-          default: { title: receiptTitle || defaultReceiptTitlePlaceholder, info: info },
```

Comment:
Here I removed the `receiptTitle || defaultReceiptTitlePlaceholder`, we don't need to default on the placeholder, if the value is empty the PDF service will just use its default